node-16.0/README.md

```diff
@@ -0,0 +1,46 @@
+# 📧 Your Function Name
+
+Brief description:
+
+<!-- For example 'Create Stripe payment and return payment URL' -->
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud functions:
+
+- **APPWRITE_FUNCTION_ENDPOINT** - Endpoint of Appwrite project
+- **APPWRITE_FUNCTION_API_KEY** - Appwrite API Key
+<!-- Add your custom environments variables -->
+
+## 🤖 Documentation
+
+_Example input:_
+
+No input
+
+<!-- If input is expected, add example -->
+
+_Example output:_
+
+<!-- Update with your expected output -->
+
+```json
+{
+    "areDevelopersAwesome": true
+}
+```
+
+## 🚀 Deployment
+
+1. Install [Appwrite CLI](todo-link-to-cli)
+2. Authenticate with `appwrite login`
+3. Open folder with your `appwrite.json`
+4. Start deploy process with `appwrite deploy`
+
+## 🎯 Configuration
+
+Intended execute access: <!-- for example 'role:all' -->
+Intended timeout: <!-- for example '60 seconds' -->
+Intended schedule: <!-- for example '0 * * * *', or 'none' -->
+Intended events: <!-- for example 'users.create' or 'none' -->
+Is intendeed to be executed through HTTP: <!-- 'yes' or 'no' -->
```

Comment:
Let's remove this section for now. doesnt seem very relevant

Reply:
Removed